UIMentoringSession/MenteesChallenges/Start Screen UI Challenge/Bilyk/MaksymBilyk_StartScreenViewController.xib

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="21701" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```

Comment:
The design in figma has 36 points to a safe area (80 to superView), but this is because of extra space. Devices without extra space (like 5s, 8plus) will have also 80 points - it will be a lot, not good

<img width="1283" alt="Screenshot 2023-05-27 at 09 27 28" src="https://github.com/alexnikol/UIMentoringSessions/assets/14887101/edf3a470-5fac-4e47-9b5e-af15e88e2513">